client/landing/stepper/hooks/use-update-flow-progress.ts

```diff
@@ -0,0 +1,136 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { Onboard } from '@automattic/data-stores';
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import type { StepPath } from '../declarative-flow/internals/steps-repository';
+
+const SiteIntent = Onboard.SiteIntent;
+const MAX_STEPS = 10;
+
+export function useUpdateFlowProgress( currentStep: StepPath, intent: string, storeType: string ) {
```

Comment:
[nit] Can we rename this to something like `useSiteSetupFlowProgress` (and rename the file as well), since we're no longer updating state within the hook?

Reply:
Updated on [useSiteSetupFlowProgress](https://github.com/Automattic/wp-calypso/blob/b68052ccf885512c5988f6fb0ede24e793cb6a8c/client/landing/stepper/hooks/use-site-setup-flow-progress.ts#L9)